hugo/content/posts/nix-bazel-crosspiling.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Nix + Bazel Crosspiling"
+date: 2023-05-24T18:00:00-07:00
+author: James Leitch
+tags:
+  - nix
+  - nixpkgs
+  - bazel
+  - bzlmod
+  - crosspile
+  - crosspiling
+  - toolchain
+  - platform
+---
+
+**Code: <https://github.com/ConsumingChaos/consumingchaos.github.io/tree/master/examples/nix-bazel-crosspiling>**
+
+# Motivation
+
+For years now, I've had "learn and set up crosspiling" as personal backlog item. From my career observations, there is often a lot of developer pain and efficiency lost when a project is targeting multiple platforms. Some examples that come to mind:
+
+- Having to switch developer machines and synchronize code across them.
+- Having to rig up together some form of individual remote compilation and teach everyone how to use it (for example, having Windows sync the repo to Mac, run the build on Mac, and report back the results).
```

Comment:
"Rig up together" is an awkward phrase. You presumably just want to drop "together."